tests/integration/relations/opensearch_provider/test_opensearch_provider.py

```diff
@@ -70,7 +70,11 @@ async def test_database_relation_with_charm_libraries(
 
     async with ops_test.fast_forward():
         # This test shouldn't take so long
-        await ops_test.model.wait_for_idle(timeout=1200, status="active")
+        await ops_test.model.wait_for_idle(apps=ALL_APPS, timeout=1200, status="active")
+
+    # await ops_test.model.block_until(
```

Comment:
Do you really need to keep this comments?

Reply:
whoops, missed this before merging. I'll remove these in my next PR.